external/webrtc-sdk.js

```diff
@@ -32703,27 +32711,20 @@ define("WebRTC_SDK/manager/RTCLocalMediaManager", [
         deferred.resolve();
         return deferred.promise;
       }
-      if (!_.isNull(stream) && !_.isUndefined(stream)) {
-        Logger.debug(
-          "RTCLocalMediaManager: " +
-            (mute ? "Muting " : "Unmuting ") +
-            "local audio stream now"
-        );
-
-        stream.getAudioTracks().forEach(function (track) {
-          track.enabled = !mute;
-        });
 
-        this.model.set("localAudioMuted", mute);
-        Logger.debug(
-          "RTCLocalMediaManager: Local Audio ",
-          mute ? "muted" : "unmuted"
-        );
-      } else {
-        Logger.warn(
-          "Local Audio stream is null, cannot mute/unmute local audio"
-        );
-      }
+      Logger.debug(
+        "RTCLocalMediaManager: " +
+          (mute ? "Muting " : "Unmuting ") +
+          "local audio stream now"
+      );
+      stream.getAudioTracks().forEach(function (track) {
+        track.enabled = !mute;
+      });
+      this.model.set("localAudioMuted", mute);
+      Logger.debug(
+        "RTCLocalMediaManager: Local Audio ",
+        mute ? "muted" : "unmuted"
+      );
```

Comment:
The diff looks weird here because we've removed a level of indentation, but this is exactly the same as [this](https://github.com/mirainc/sdk-webrtc-meetings/pull/4/files#diff-7c9d62cf9e59419121e70349e9a94586L32707-L32721)